### PR TITLE
Refactor payouts table for web bug bounty program

### PR DESCRIPTION
**Pull request by fkiriakos07**
## One-line summary

This changeset refactors the payouts table for the web bug bounty program in order to simplify bounty decisions and make them more consistent.
 
Proposed changes are documented here: https://docs.google.com/document/d/1uTSekST5nOuwo6EKDFhDy_OORH1Cdh1msCLDe_-m4LQ/edit

## Significant changes and points to review

- Specify payouts based on vulnerability severity rating instead of vulnerability class
- use payout ranges instead of specific amount
- add a table to define each severity rating and give examples

## Screenshots
image

Screenshot 2022-09-05 at 16 22 41

### CODE REVIEW

**Review by gene1wood — APPROVED** [review]

**Comment by fkiriakos07** [comment]
@dveditz I removed the confusing screenshot, and the bounty ranges are listed here: https://docs.google.com/document/d/1uTSekST5nOuwo6EKDFhDy_OORH1Cdh1msCLDe_-m4LQ/edit

**Comment by craigcook** [comment]
Let us know when this is ready.

**Review by dveditz — APPROVED** [review]
lgtm +1

**Comment by fkiriakos07** [comment]
Hello @craigcook this is ready for your review, thank you

File: bedrock/security/templates/security/web-bug-bounty.html [hunk 1]
```diff
@@ -27,102 +27,123 @@ <h2 id="payouts-section">Payouts</h2>
...
+        <td>
+          <p>
+            Critical vulnerabilities are urgent security issues that present an ongoing or immediate danger to the users of our services and our infrastructure
+            <ul class="mzp-u-list-styled">
+              <li>Remote Code Execution</li>
+              <li>Authentication and Session Management Flaws (which lead to account compromise)</li>
+              <li>Disclosure of secrets in publicly accessible assets</li>
+              <li>Hardcoded credentials for a privileged user</li>
+            </ul>
+          </p>
```
Comment (craigcook) [inline bedrock/security/templates/security/web-bug-bounty.html:91]: Nit: this closing `</p>` tag should go before the list. The same for the other three below.

**Review by craigcook — CHANGES_REQUESTED** [review]

File: bedrock/security/templates/security/web-bug-bounty.html (hunk 1 above)
Comment (fkiriakos07) [reply bedrock/security/templates/security/web-bug-bounty.html:85]: done, thanks

**Review by craigcook — APPROVED** [review]

**Pull request merged**